src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,31 +128,34 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        //ensuring same mode of operation is used when the data is less than 1 block long
+        String plaintextLength28 = StringHelper.generateStringOfLength(7, 'A');
+        assertEncryptString(plaintextLength28);
+
+        //ensuring same mode of operation is used when the data is equal to 1 block
+        String plaintextLength128 = StringHelper.generateStringOfLength(32, 'A');
+        assertEncryptString(plaintextLength128);
+
+        //check if padding is applied, ensure same mode of operation is used when the data is more than 1 block long
```

Comment:
In this case we are calling the same `assert*` method as the previous two cases yet the comment is different. Perhaps limit the comment to describe the scenario rather than the expected outcome?  e.g. `// Data is more than 1 block long`